src/Migrations/20191230143918-create-disk-information.ts

```diff
@@ -0,0 +1,39 @@
+import {QueryInterface, DataTypes} from 'sequelize';
+
+module.exports = {
+    up: (queryInterface: QueryInterface) => {
+        return queryInterface.createTable('DiskInformations', {
```

Comment:
This is funny as there is no plural for information :) You can use `freezeTableName: true` and use it as `DiskInformation`

Reply:
After changes, delete db data and run the migrations again and test it to make sure everything still works.